EasyPost.Tests/BatchTest.cs

```diff
@@ -130,10 +131,14 @@ public async Task TestCreateScanForm()
 
 
             Batch batch = await CreateOneCallBuyBatch();
+
+            // Uncomment the following line if you need to re-record the cassette
+            // Thread.Sleep(10000); // Wait enough time to process
```

Comment:
Let's remove this first occurrence of the sleep timer. There is no need to sleep between creation and buying.